packages/esm-form-entry-app/src/app/form-data-source/form-data-source.service.ts

```diff
@@ -303,7 +303,21 @@ export class FormDataSourceService {
 
   public getPatientObject(patient): object {
     const model: object = {};
-    model['sex'] = patient?.gender === 'male' ? 'M' : 'F';
+    const gender = patient?.gender;
+    model['sex'] = (gender) => {
+      switch (gender) {
+        case 'male':
+          return 'M';
+        case 'female':
+          return 'F';
+        case 'other':
+          return 'O';
+        case 'unknown':
+          return 'U';
+        default:
+          return 'M';
```

Comment:
Why is the default gender Male? Realistically, you can't an individual's gender until he/she tell you.